<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QQMusicDemo</class>
 <widget class="QWidget" name="QQMusicDemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>690</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QQMusicDemo</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>qrc:/images/QQMusic.png</normaloff>qrc:/images/QQMusic.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QQMusicMenu" name="wMenu" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>65</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>65</height>
      </size>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QQMusicCenterW" name="wCenter" native="true">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QQMusicListW" name="wList" native="true">
          <property name="minimumSize">
           <size>
            <width>210</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>210</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QStackedWidget" name="stackedWidget">
            <widget class="QWidget" name="page"/>
            <widget class="QWidget" name="page_2"/>
           </widget>
          </item>
          <item>
           <widget class="QQMusicPlay" name="wPlayer" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>78</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQMusicMenu</class>
   <extends>QWidget</extends>
   <header>./src/qqmusicmenu.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QQMusicCenterW</class>
   <extends>QWidget</extends>
   <header>./src/qqmusiccenterw.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QQMusicPlay</class>
   <extends>QWidget</extends>
   <header location="global">./src/qqmusicplay.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QQMusicListW</class>
   <extends>QWidget</extends>
   <header location="global">./SRC/qqmusiclistw.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
